Source/WebKit/UIProcess/Extensions/WebExtensionMatchPattern.h

```diff
@@ -77,6 +79,9 @@ class WebExtensionMatchPattern : public API::ObjectImpl<API::Object::Type::WebEx
 
     static void registerCustomURLScheme(String);
 
+    static void matchPatternStrings(const MatchPatternSet&, HashSet<String>&);
+    static void matchPatternsForOrigins(HashSet<String>&, MatchPatternSet&);
```

Comment:
Ditto, this can return via `MatchPatternSet&&`. Maybe call this `MatchPatternSet&& fromStrings(HashSet<String>&)`